Configure.ps1

```diff
@@ -4,8 +4,8 @@
 # Before running this script you need to install the Azure RM cmdlets as an administrator. 
 # For this:
 # 1) Run Powershell as an administrator
-# 2) in the PowerShell window, type: Install-Module AzureRM.Resources
-#
+# 2) in the PowerShell window, type: Install-Module AzureAD
+
 # To prepare this script
 # 3) With the Azure portal (https://portal.azure.com), choose your active directory tenant, then go to the Properties of the tenant and copy
 #    the DirectoryID. This is what we'll use in this script for the tenant ID
```

Comment:
In Powershell, do not call functions using paranthesis - i.e. cleanup($apps)  on line 22. Instead, call it with cleanup $apps. Also, to be consistent with the other functions - please use Cleanup.